.github/workflows/validate-schema.yml

```
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  # Allow manual trigger
+  workflow_dispatch:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)